src/Components/ProjectForm.js

```diff
@@ -77,6 +92,14 @@ function ProjectForm() {
   const [message, setMessage] = React.useState("");
   const [user, setUser] = React.useState([""]);
   const [tags, setTags] = React.useState("");
+  const [country, setCountry] = React.useState("");
+  const [funding, setFunding] = React.useState("");
+
+  const history=useHistory();
+
+  const handleRoute = () =>{ 
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8488139)
```suggestion
  const handleRoute = () => { 
```